src/app/components/mnoe-api/admin/organizations.svc.coffee

```diff
@@ -35,8 +35,14 @@
   @inArrears = () ->
     MnoeAdminApiSvc.all('organizations').all('in_arrears').getList()
 
+  # Cache a single organization
+  organization = {}
   @get = (id) ->
-    MnoeAdminApiSvc.one('organizations', id).get()
+    if organization[id]
+      organization[id]
+    else
+      organization[id] = MnoeAdminApiSvc.one('organizations', id).get()
```

Comment:
Might be a little cleaner refactor:
```
@get = (id) ->
  organization[id] ?= MnoeAdminApiSvc.one('organizations', id).get()
```
`?=` in coffeescript is similar to `||=` in ruby